docs/styling-conventions.md

```diff
@@ -1,14 +1,156 @@
 # Component Styling Conventions
 
 ## Summary
-Components must use the [styled-components](https://www.styled-components.com/) package to style all HTML elements they render, and allow certain styles to be specified by a [styled-components theme](https://www.styled-components.com/docs/advanced#theming). Add the default values for any theme properties you use in /src/defaultComponentTheme.js. All theme properties must start with `rui_` followed by a camelcased identifier.
+Components must use the [styled-components](https://www.styled-components.com/) package to style all HTML elements they render, and allow certain styles to be specified by a [styled-components theme](https://www.styled-components.com/docs/advanced#theming).
 
-By convention visual and text styles should be baked into each component. This means that a containing element should set the font-family, font-weight, font-size etc. This convention will make it easier to integrate components into the Reaction Starter Kit and other environments. Further, style values should be retrieved from the theme as seen in the example below.
+## Theme Variables
 
+The default styled-components theme, which is used in the style guide and baked into the components package, is in `/src/defaultComponentTheme.js`.
+
+The following styles should almost always be put into the theme file as variables, rather than hard coded in the component file:
+- padding
+- colors
+- border width
+- border radius
+- font family
+- font size
+- font weight
+
+> ALWAYS use the most specific CSS rule there is. For example, you should never have just a single "Padding" variable, but rather "PaddingLeft", "PaddingRight", "PaddingTop", and "PaddingBottom". Similarly, never use just "Border" or "BorderLeft", but rather the full specific rule such as "BorderLeftColor" or "BorderTopLeftRadius".
```

Comment:
Does this mean... if we're adding a default `10px` padding to all sides, we should declare:
```
PaddingLeft: 10px,
PaddingRight: 10px,
PaddingTop: 10px,
PaddingBottom: 10px
```

Right?